### ISSUE TITLE
Error with loadlocale: _nl_intern_locale_data

### ISSUE THREAD

**Issue by dariober** [EBIvariation/vcf-validator#184]
Hi- with vcf_validator v0.9.1 on Ubuntu 18.04 I get the error:

```
./vcf_validator -i data/common.sample.vcf.gz
[info] Reading from input file...
vcf_validator: loadlocale.c:129: _nl_intern_locale_data: Assertion `cnt < (sizeof (_nl_value_type_LC_TIME) / sizeof (_nl_value_type_LC_TIME[0]))' failed.
Aborted (core dumped)
```

After some googling I found I can work around it by first executing  `export LC_ALL=C` but it would be nice to have a more permanent solution...

**Comment by jmmut** [comment]
Hi, yes, you are right, we noticed this some days ago and reached to the same conclusion. However, we didn't have time to fix this yet. I can put a short notice in the readme for now. We will update and close this issue when it's fixed. Thanks.

**Comment by jmmut** [comment]
This is from april... 6 months already... Be sure I haven't forgotten about this. I'll try a bit harder to convince my bosses to let me (or someone else) spend some time on this. If there's a PR from anyone I'll probably take a look in my personal spare time.

**Comment by jmmut** [comment]
fixed in the latest release: v0.9.3: https://github.com/EBIvariation/vcf-validator/releases/tag/v0.9.3
